### ISSUE TITLE
Include not working

### ISSUE THREAD

**Issue by ThunderSphun** [Settis/dasm-vscode#50]
How do I use include with this plugin, as I am working on a project, where include is succesfully used, but this extension doesnt find the files. the project can be build correctly, however, all includes are ignored due to errors, causing many more errors to appear.
I dont know if this has something to do with how I use include, but seeing as dasm can build the project without a problem, I assume I am doing it correctly
Whether I use `include file.asm` or `include "file.asm"`, it gives errors. both build fine
`include file.asm` gives the error: `Redundant input, expecting EOF but found: .asm` `parser`
`include "file.asm"` gives the error: `The file is not resolvable`
Where is it searching for the files to include, and why does it need to be a string

**Comment by Settis** [comment]
I see two issues there.
- `include file.asm` gives an error. Didn't know that this is valid for dasm, I use string literals instead. I'll fix it as a separate bug #51.
- `include "file.asm"` also gives an error. This part is not clear. Can you give more examples on it?

The plugin tries to find files in the same directory as the current file is opened. It's not necessery the root of the project or currently opened folder in VS Code.

**Comment by ThunderSphun** [comment]
the folder structure is like this:
```
source
    main.asm
    macros.asm
build.nes
```

`main.asm` contains the line `			include macros.asm` it doesn't find the file `source/macros.asm`, with the error message that I gave yesterday (`The file is not resolvable`)

ps.
tried this in the root folder and the redundant input was given, like when not using a string literal

**Comment by Settis** [comment]
I understand the cause of the error with redundant input. And it's clear how to fix it.
I'm afraid that there is another bug in the file-resolving part. So after fixing the `redundant input` you will have `The file is not resolvable` in both cases (string literal and name without quotes).
I did a quick test with these files:
```
> cat source/main.asm 
    include "macros.asm"
    JMP foo
> cat source/macros.asm 
foo:
>
```
There are no errors here. The label can be resolved correctly.

I've tested the plugin on Linux only. Do you use Windows?

**Comment by ThunderSphun** [comment]
yes, i am using windows

**Comment by Settis** [comment]
I see...
Well, at least I have an idea of where to dig.

**Comment by Settis** [comment]
It must be fixed in 1.1.3 version.

**Comment by ThunderSphun** [comment]
yes, this seems to do it
